src/js/hca-rjsf/config/form.js

```diff
@@ -193,7 +195,9 @@ const formConfig = {
         demographicInformation: {
           path: 'veteran-information/demographic-information',
           title: 'Veteran information',
-          initialData: {},
+          initialData: {
+            isSpanishHispanicLatino: false
```

Comment:
This one is actually intentional! For some reason, the backend schema requires this field. So, I'm defaulting it to false (which causes no visual changes, because it's a checkbox) to get around that based on Mark's suggestion.